tutorials/onchain-governance/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: On-Chain Governance Tutorials
+description: Learn how to utilize Polkadot OpenGov with step-by-step tutorials on on-chain governance, including proposals, referenda, delegation, and voting processes.
+hide: 
+- feedback
+template: index-page.html
+---
+
+# On-Chain Governance Tutorials
+
```

Comment:
We should add some content here

Reply:
Yes agree with the above. Not sure why this was marked as resolved or if some conclusion was made here that I missed